tox.ini

```diff
@@ -5,7 +5,6 @@ skipsdist = True
 
 [testenv]
 usedevelop = true
-install_command = pip install -c{toxinidir}/constraints.txt -U {opts} {packages}
```

Comment:
Why is this entire command removed?

Reply:
Missed the `install_command` syntax in [tox](https://tox.wiki/en/latest/config.html) - fixed now